src/index.ts

```diff
@@ -39,6 +39,7 @@ import {
   ValidatorAction,
   ValidatorMap
 } from './types';
+import { isArrayObject } from './utils/array-object';
```

Comment:
✂️ 

Should make sure this is caught by linting in CI

Reply:
I can PR tomorrow that adds / updates linting in the CI config tomorrow?